### PR TITLE
Fix setting of S3 XML document namespace

### PR DESCRIPTION
**Pull request by michal-josef-spacek**
Issue is in fact, that AWS and GCS returns different XML namespace.
AWS returns:
`<ListBucketResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/">`
GCS returns:
`<ListBucketResult xmlns='http://doc.s3.amazonaws.com/2006-03-01'>`

Signed-off-by: Michal Josef Špaček <michal.spacek@gooddata.com>

### CODE REVIEW

**Comment by tupinek** [comment]
I missed running of test, so i am working on it again.

File: t/api-bucket-objects-list.t [hunk 1]
```diff
@@ -153,3 +158,34 @@ expect_api_bucket_objects_list 'error no such bucket' => (
     expect_s3_errstr        => 'No such bucket error message',
 );
 
+expect_api_bucket_objects_list 'list objects on Google Cloud Storage (version 1)' => (
```
Comment (happy-barney) [inline t/api-bucket-objects-list.t]: Please small change in test case description. version 1 relates to S3 API call not to service

```suggestion
expect_api_bucket_objects_list 'list objects (version 1) on Google Cloud Storage' => (
```

Comment (tupinek) [reply t/api-bucket-objects-list.t]: done

**Comment by ranguard** [comment]
@happy-barney ok to merge?

**Comment by happy-barney** [comment]
yes let's merge it

**Comment by michal-josef-spacek** [comment]
Could we merge this?

**Pull request merged**